gravatar/api/gravatar.api

```diff
@@ -322,8 +322,9 @@ public final class com/gravatar/restapi/models/CryptoWalletAddressKt {
 }
 
 public final class com/gravatar/restapi/models/GalleryImage {
-	public synthetic fun <init> (Ljava/net/URI;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Ljava/net/URI;Ljava/lang/String;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
**TIL** - In spite of the fact that we don't have a public constructor, I was worried about this change in the method so I've asked Gemini:

> A constructor declared as public synthetic fun <init> in a .api file would not be directly accessible from your code.
>
>The keyword **synthetic** indicates that this constructor is not a real part of the original code. It's generated by the compiler for internal purposes, often related to how Kotlin interacts with Java bytecode.